workflow-archiver/workflow_archiver/version.txt

```diff
@@ -1 +1 @@
-0.2.4
+0.2.5
```

Comment:
Should we just change this to 0.6.1 it's very confusing to me why workflow archiver has a different version from all the other packages

Reply:
I agree with you @msaroufim .  @lxning Is that fine?

Reply:
we need completely redesign workflow soon. So I don't it is necessary to change at this moment.